p2p/libp2p/identity/crypto.go

```diff
@@ -0,0 +1,24 @@
+package identity
+
+import (
+	"encoding/hex"
+
+	"github.com/libp2p/go-libp2p/core/crypto"
+	poktCrypto "github.com/pokt-network/pocket/shared/crypto"
+)
+
+// NewLibP2PPrivateKey converts a hex-encoded ed25519d key
```

Comment:
Should this be under `p2p` or under `shared/crypto` in your opinion?

Reply:
I don't think it really belongs in the "identity" concern but I was reluctant to put things into `shared/crypto` as it wasn't clear at the time of writing what pieces of code would be ephemeral and/or P2P specific. I think as it's currently written it would make more sense to move it. Thanks for calling that out! :raised_hands: